examples/0001-overview.qmd

```diff
@@ -7,41 +7,41 @@ The examples cover various aspects, including authentication, data retrieval, an
 ## Example - Print information for each user where `user_role='publisher'`
 
 ```python
-from posit.connect import Client
+from posit.connect import Connect
 
-with Client() as client:
-    for user in client.users.find(user_role='publisher'):
+with Connect() as connect:
```

Comment:
maybe `with Connect() as con` or something, just to disambiguate? (Or is that more ambiguous?)